python/src/energuide/extracted_datatypes.py

```diff
@@ -496,3 +583,215 @@ def to_dict(self) -> typing.Dict[str, typing.Union[str, float]]:
             'airFlowRateCfm': self.air_flow_rate_cmf,
             'efficiency': self.efficiency,
         }
+
+
+class WaterHeating(_WaterHeating):
+
+    _TYPE_MAP = {
+        ("Electricity", "Not applicable"): (
+            WaterHeaterType.NOT_APPLICABLE.value,
```

Comment:
Usually you just point to the `WaterHeaterType. NOT_APPLICABLE ` instead of the value of the enum.